File: VI семестр/ППКС - Програмування для паралельних комп_ютерних систем/6_java/Матвійчук java/pro/src/lab3/matrix.java
package lab3;

public class matrix {
	/**
	 * матриця int
	 */
	private int[][] arr;
	/**
	 * розмір матриці mas[n][m]
	 */
	private int n;
	/**
	 * розмір матриці mas[n][m]
	 */
	private int m;
	/**
	 * @param n розмір матриці mas[n][m]
	 * @param m розмір матриці mas[n][m]
	 */
	public matrix(int n, int m) {
		arr = new int [n][m];
		this.n=n;
		this.m=m;
	}
	/** задає значення вказаной перемінной
	 * @param n номер позиції  mas[n][m]
	 * @param m номер позиції  mas[n][m]
	 * @param val перемінна int
	 */
	public void set(int n, int m, int val) {
		arr[n][m]=val;
	}
	/**
	 *  заповнює матрицю 1
	 */
	public void set1() {
		for (int i=0;i<n;i++){
			for (int j=0;j<m-1;j++){
				arr[i][j]=1;
			}
		}
	}
	/** вертає перемінну 
	 * @param index n номер позиції перемінной  mas[n][m]
	 * @param index m номер позиції перемінной  mas[n][m]
	 * @return int
	 */
	public int get(int a, int b) {
		return arr[a][b];
	}
	
	/** розмір матриці
	 * @return int
	 */
	public int getSize() {
		return arr.length;
	}
	
	/** формування строкі 
	 * @return String
	 */
	public String toString() {
		String res = "";
		for (int i=0;i<n;i++){
			for (int j=0;j<m;j++){
				res+=Integer.toString(arr[n][m]);
			}
			res+='\n';
		}
		return res;
	}
}

File: VI семестр/ППКС - Програмування для паралельних комп_ютерних систем/3_c_sharp/Кривоносов csharp/source/Lab3/Lab3/Matrix.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
/**
 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
 *                                                             *
 *            Parallel and Distributed Computing               *
 *             Laboratory work #3. C#                          *
 *                                                             *
 * Task: A=(B*C)*Z+α*E*(MX*MZ)                                 *
 *                                                             *
 *                                                             *
 * @author Krivonosov Oleksii                                  *
 * @group IO-34                                                *
 * @date 24.03.16                                              *
 * File Matrix.cs                                              *
 *                                                             *
 * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
 */

namespace lab3
{
    class Matrix
    {
        private Vector[] vector;

        public Matrix(int n)
        {
            vector = new Vector[n];
            for (int i = 0; i < vector.Length; i++)
            {
                vector[i] = new Vector(n);
            }
        }

        public void set(int n, int m, int val)
        {
            vector[n].set(m, val);
        }

        public int get(int n, int m)
        {
            return vector[n].get(m);
        }

        public Vector get(int index)
        {
            return vector[index];
        }

        public int size()
        {
            return vector.Length;
        }

        public String toString()
        {
            String res = "";
            for (int i = 0; i < vector.Length; i++)
            {
                res += vector[i].toString();
                if (i != vector.Length - 1)
                {
                    res += "\n";
                }
            }
            return res;
        }


    }
}
